### PR TITLE
DROOLS-4395 [DMN Designer] Validation fails for included PMML model

### PR DESCRIPTION
**Pull request by tarilabs**

### CODE REVIEW

File: kie-dmn/kie-dmn-validation/src/test/java/org/kie/dmn/validation/ValidatorImportTest.java [hunk 1]
```diff
@@ -200,4 +203,26 @@ public void testImportPMML() throws IOException {
         assertThat(ValidatorUtil.formatMessages(messages), messages.size(), is(0));
 
     }
+
+    @Test
+    public void testImportPMML2() throws IOException {
```
Comment (tarilabs) [inline kie-dmn/kie-dmn-validation/src/test/java/org/kie/dmn/validation/ValidatorImportTest.java:208]: @karreiro @manstis kindly advise if this addresses Option2 as discussed, please?
If yes I will move the PR from draft into to-be-reviewed and eventually merged.

**Comment by tarilabs** [comment]
Provisional feedback from WB side is positive, @baldimir @jiripetrlik for your code review please.

File: kie-dmn/kie-dmn-validation/src/test/java/org/kie/dmn/validation/ValidatorImportTest.java (hunk 1 above)
Comment (manstis) [reply kie-dmn/kie-dmn-validation/src/test/java/org/kie/dmn/validation/ValidatorImportTest.java:208]: Looks good to me, abstracting provision of imports themselves to the user of `DMNValidator`.

**Review by jiripetrlik — APPROVED** [review]
Looks good to me.

**Pull request merged**